dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat_metrics/test-applications/CDIFaultToleranceMetrics.war/resources/META-INF/permissions.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2017 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+
+<permissions xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/permissions_7.xsd"
+	version="7">
+        <!-- this permission shouldn't be needed but there is a java bug -->
+        <permission>
```

Comment:
Is this java bug workaround still needed?

Reply:
I don't know but we should check so I will open an issue to do so.